src/api/index.ts

```diff
@@ -1 +1,21 @@
-export * from "./client";
+export { createZnsApiClient, znsApiClient } from "./znsApi/client";
+export { createDataStoreApiClient, DataStoreApiClient as DataStoreApiClient } from "./dataStoreApi/client";
+// import { znsApiClient } from "./znsApi/client";
```

Comment:
remove code?

Reply:
Was trying to do something more configurable but went with this instead, removed